server/channels/store/sqlstore/channel_store.go

```diff
@@ -124,6 +152,30 @@ func channelMemberToSlice(member *model.ChannelMember) []any {
 	return resultSlice
 }
 
+func channelToSlice(channel *model.Channel) []interface{} {
```

Comment:
Would this naming be more clear?
```suggestion
func channelToValueSlice(channel *model.Channel) []interface{} {
```

Reply:
We already have a lot of similar function with the pattern `<entity>ToSlice`, so keeping it this way for consistency.